ui/send_page.go

```diff
@@ -260,6 +305,9 @@ func (pg *SendPage) setAccountTabs() {
 }
 
 func (pg *SendPage) Layout(common pageCommon) {
+	go pg.getUSDValues(&pg.data)
```

Comment:
This method is going to be called repeatedly, and even though it's called from a go routine for optimal reasons that's still a lot of go routines to spin up. Wouldn't it be better if this was only called when a user types the amount into the amount field?

Reply:
The idea is to ensure the exchange is readily available at the point of typing the input and also not to make the app to freeze while trying to get the exchange data.

Reply:
Removing to go routine and moving it to the calculatedvalue function.

Reply:
> The idea is to ensure the exchange is readily available at the point of typing the input and also not to make the app to freeze while trying to get the exchange data.

Still, calling an endpoint that frequent isn't an optimal solution to that. If real time time updates is what is needed, using sockets would be a better approach. Recursive calls to an endpoint, is a No No. Plus I don't think the exchange rates change that frequently to be called multiple times in a fraction of a second.